### PR TITLE
[ADD] l10n_ar_ux: Data on activities, taxes and concepts is create

### PR DESCRIPTION
**Pull request by mem-adhoc**
Task: 28319

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/293).

File: l10n_ar_edi_ux/models/res_partner.py [hunk 1]
```diff
@@ -87,8 +87,44 @@ def get_data_from_padron_afip(self):
...
+
         actividades = [str(act["idActividad"])
                        for act in data_rg.get("actividad", []) + data_mt_actividades]
+
+        def check_activity(data_rg):
+            res = []
+            new_activity = {}
+            afip_activities = data_rg.get('actividad')
+            for act in afip_activities:
+                if str(act.get('idActividad')) not in (self.actividades_padron.search([]).mapped('code')):
```
Comment (zaoral) [inline l10n_ar_edi_ux/models/res_partner.py]:
```suggestion
                if str(act.get('idActividad')) not in activity_codes:
```

File: l10n_ar_edi_ux/models/res_partner.py [hunk 2]
```diff
@@ -87,8 +87,44 @@ def get_data_from_padron_afip(self):
...
             data_mt_actividades = [data_mt_actividades]
 
+
         actividades = [str(act["idActividad"])
                        for act in data_rg.get("actividad", []) + data_mt_actividades]
+
+        def check_activity(data_rg):
+            res = []
+            new_activity = {}
+            afip_activities = data_rg.get('actividad')
```
Comment (zaoral) [inline l10n_ar_edi_ux/models/res_partner.py:96]: activity_codes = self.actividades_padron.search([]).mapped('code')

File: l10n_ar_edi_ux/models/res_partner.py [hunk 3]
```diff
@@ -87,8 +87,44 @@ def get_data_from_padron_afip(self):
...
+                    res.append(act)
+            return res
+        check_activity(data_rg)
+
+        def check_taxes(data_mt, data_rg):
+            res = []
+            new_tax = {}
+            afip_taxes = data_mt.get("impuesto", []) + data_rg.get("impuesto", [])
+            for imp in afip_taxes:
+                if str(imp.get('idImpuesto')) not in (self.impuestos_padron.search([]).mapped('code')):
```
Comment (zaoral) [inline l10n_ar_edi_ux/models/res_partner.py]: same of above

**Review by zaoral — CHANGES_REQUESTED** [review]

**Review by zaoral — APPROVED** [review]

**Comment by zaoral** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#313

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#313

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#313

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#313

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#313

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#313

**Pull request closed without merging**